src/tasks/deploy-utils.js

```diff
@@ -66,9 +66,18 @@ function validateId(settings) {
  * @memberof slate-cli.tasks.watch, slate-cli.tasks.deploy
  * @private
  */
-
 gulp.task('validate:id', () => {
-  const file = fs.readFileSync(config.tkConfig, 'utf8');
+  let file;
+
+  try {
+    file = fs.readFileSync(config.tkConfig, 'utf8');
+  } catch (err) {
```

Comment:
Update why don't you use `existsSync`? Only `exists` was deprecated. (https://nodejs.org/api/fs.html#fs_fs_existssync_path)